apps/emqx_schema_validation/src/emqx_schema_validation.erl

```diff
@@ -199,6 +204,60 @@ post_config_update(?VALIDATIONS_CONF_PATH, {reorder, _Order}, New, _Old, _AppEnv
     ok = emqx_schema_validation_registry:reindex_positions(New),
     ok.
 
+%%------------------------------------------------------------------------------
+%% `emqx_config_backup' API
+%%------------------------------------------------------------------------------
+
+import_config(#{?CONF_ROOT_BIN := RawConf0}) ->
+    OldRawValidations = emqx_config:get_raw([?CONF_ROOT_BIN, <<"validations">>], []),
+    {ImportedRawValidations, RawConf1} =
+        case maps:take(<<"validations">>, RawConf0) of
+            error ->
+                {[], RawConf0};
+            {V, R} ->
+                {V, R}
+        end,
+    %% If there's a matching validation, we don't overwrite it.  We don't remove any
+    %% validations, either.
+    #{added := NewRawValidations} = emqx_utils:diff_lists(
+        ImportedRawValidations,
+        OldRawValidations,
+        fun(#{<<"name">> := N}) -> N end
+    ),
+    OtherConfs = maps:to_list(RawConf1),
+    Result = emqx_utils:foldl_while(
+        fun
+            ({validation, RawValidation}, Acc) ->
+                case insert(RawValidation) of
```

Comment:
this generates small config changes,
would it make sense to merge all configs and reload all at once?

Reply:
:heavy_check_mark: